src/token/erc721/erc721.cairo

```diff
@@ -164,12 +191,17 @@ mod ERC721 {
             self._approve(to, token_id);
         }
 
+        /// Enable or disable approval for `operator` to manage all of the
+        /// caller's assets.
+        /// Emits an [Approval](Approval) event.
```

Comment:
```suggestion
        /// Enable or disable approval for `operator` to manage all of the
        /// caller's assets.
        ///
        /// Emits an [Approval](Approval) event.
```

Solidity's library is doing this, and I think it reads better. I would separate the short description, from the long one, and from the requirements, and events, etc... with a line.